### DISCUSSION TITLE
所有消息都已读后还是会显示有 8 条未读的“我的消息”

### DISCUSSION THREAD

**Discussion by HPDell** [BewlyBewly/BewlyBewly#739] [General]
即使所有消息都已读，顶栏也会如下图显示

![image](https://github.com/BewlyBewly/BewlyBewly/assets/17842185/5a7c25e0-5b9c-4e2b-a5f9-60842e0f1cdb)

但如果关掉插件，则显示没有未读消息

![image](https://github.com/BewlyBewly/BewlyBewly/assets/17842185/d1e2a202-9950-443b-b5dd-a6f03ec28ee5)

**Comment by hakadao** [comment]
沒有吧現在最新版的話

**Reply by HPDell** [reply]
没了，感谢！
